modin/_compat/pandas_api/latest/base.py

```diff
@@ -15,17 +15,32 @@
 
 import pandas
 from pandas.core.dtypes.common import is_datetime_or_timedelta_dtype
+from pandas.core.window.ewm import ExponentialMovingWindow
 from pandas.util._validators import validate_bool_kwarg, validate_ascending
-from pandas._libs.lib import no_default
-from pandas._typing import StorageOptions, CompressionOptions, IndexLabel
+from pandas._libs.lib import no_default, NoDefault
+from pandas._typing import (
+    StorageOptions,
+    CompressionOptions,
+    IndexLabel,
+    TimedeltaConvertibleTypes,
+    Axis,
+)
 import pickle as pkl
-from numpy import nan
-from typing import Sequence, Hashable, Optional
+import numpy as np
+from typing import Sequence, Hashable, Optional, TYPE_CHECKING
 
 from ..abc import BaseCompatibleBasePandasDataset
 from .utils import create_stat_method
 from modin.utils import _inherit_docstrings
 
+if TYPE_CHECKING:
+    from modin.pandas.base import BasePandasDataset
+
+nan = np.nan
+
+# See https://github.com/pandas-dev/pandas/blob/v1.4.3/pandas/core/generic.py#L195
```

Comment:
for future-proofing, should this link to the permalink? i am not aware of another way to link (symbolically would be nice). https://github.com/pandas-dev/pandas/blob/e8093ba372f9adfe79439d90fe74b0b5b6dea9d6/pandas/core/generic.py#L195

Reply:
I guess since this is pinned to 1.4.3, it's unlikely to change since pandas doesn't backport afaik. Feel free to ignore.